recipe/meta.yaml

```diff
@@ -1,42 +1,52 @@
 {% set name = "apeye-core" %}
-{% set version = "1.1.0" %}
+{% set version = "1.1.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/apeye_core-{{ version }}.tar.gz
-  sha256: b57953dbd13d6e6c6228281bbf4468b98bd21261f1b23812bfab811f247edc0a
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_')}}-{{ version }}.tar.gz
+  sha256: 5de72ed3d00cc9b20fea55e54b7ab8f5ef8500eb33a5368bc162a5585e238a55
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<36]
 
 requirements:
   host:
-    - hatch-requirements-txt
     - pip
-    - python >=3.6
+    - python
+    - wheel
+    - setuptools
```

Comment:
Does it rely on hatchling instead? See https://github.com/domdfcoding/apeye-core/blob/v1.1.5/pyproject.toml#L3